### ISSUE TITLE
Apple-Letter navigation does not work with filenames that start with a lowercase letter

### ISSUE THREAD

**Issue by c0mmander8** [a2stuff/a2d#615]
**Describe the bug**
Apple-Letter key keyboard navigation does not work with files or folders with GS/OS (ProDOS 2.5.x) or AppleWorks filenames that start with a lowercase letter or any file or folder located after them in alphabetical order in the file picker dialog boxes, both in the Apple II DeskTop app and in the Selector app.

**To Reproduce**
1. Start Apple II DeskTop with the startup disk containing files and/or folders with GS/OS (ProDOS 2.5.x) filenames starting with the following letters: capital "A", capital "B", lowercase "c", capital "D" and capital "E".
2. Click on the File menu and go to File > Copy a File....
3. Make sure that the startup disk is selected.
4. Type Apple-A.
5. The file or folder with a GS/OS (ProDOS 2.5.x) filename starting with a capital "A" is highlighted.
6. Type Apple-B.
7. The file or folder with a GS/OS (ProDOS 2.5.x) filename starting with a capital "B" is highlighted.
8. Type Apple-C.
9. The file or folder with a GS/OS (ProDOS 2.5.x) filename starting with a lowercase "c" is not highlighted and the file or folder with a GS/OS (ProDOS 2.5.x) filename starting with a capital "B" is still highlighted.
10. Type Apple-D.
11. The file or folder with a GS/OS (ProDOS 2.5.x) filename starting with a capital "D" is not highlighted and the file or folder with a GS/OS (ProDOS 2.5.x) filename starting with a capital "B" is still highlighted.
12. Type Apple-E.
13. The file or folder with a GS/OS (ProDOS 2.5.x) filename starting with a capital "E" is not highlighted and the file or folder with a GS/OS (ProDOS 2.5.x) filename starting with a capital "B" is still highlighted.

**Expected behavior**
Apple-Letter key keyboard navigation should work with files or folders with GS/OS (ProDOS 2.5.x) or AppleWorks filenames that start with a lowercase letter and any file or folder located after them in alphabetical order in the file picker dialog boxes, both in the Apple II DeskTop app and in the Selector app.

**Screenshots**
 I am sorry. I have no screenshots to help explain my problem.

**System Details (please complete the following information):**
 - System [e.g. Apple IIe Enhanced, Apple IIc, Apple IIgs] Apple IIGS (ROM 01)
 - Emulator? [e.g. real hardware, or emulator name] GSplus 0.14 (the latest version of the program)
 - ProDOS version ProDOS 2.4.2 (the latest version)
 - DeskTop version (from About > Apple II DeskTop...) Apple II DeskTop 1.2 Alpha 40 (the latest version of the program)
 - Other [e.g. mouse, memory expansions, disk types, etc]

**Additional context**
 There is no other context about the problem here.
